packages/poi-preset-elm/index.js

```diff
@@ -0,0 +1,22 @@
+module.exports = options => {
+  return poi => {
+    const optionDefaults = {
+      pathToMake: 'node_modules/.bin/elm-make',
+      warn: true,
+      debug: true
+    }
+    poi.extendWebpack(config => {
+      config.module
+        .rule('elm')
+          .test(/\.elm$/)
+          .exclude.add([/elm-stuff/, /node_modules/]).end()
+          .use('elm-hot')
+            .loader('elm-hot-loader').end()
+          .use('elm-webpack')
+            .loader('elm-webpack-loader')
+              .options(poi.options.mode === 'production' ? {} : Object.assign(optionDefaults, options || {}))
```

Comment:
why do we only set `options` in non-production mode?

Reply:
Hmm, yeah we do want some of them. Committing a change that will only disable debug and leave the others as is.

Reply:
I appreciate how thorough you are with these reviews ;)

Reply:
Hmm.. odd. New commits not showing up.

Reply:
Only when I force pushed they appeared.